docs/tutorials/control/air-quality-fleet.md

```diff
@@ -0,0 +1,809 @@
+
+    {{<imgproc src="/tutorials/air-quality-fleet/synced-data.png" resize="x1100" declaredimensions=true alt="The sensor readings that have synced to the DATA page." style="max-width:600px" >}}
+
+Once you've confirmed that data is being collected and synced correctly, you're ready to start building a dashboard to display the data.
+If you'd like to graph your data using a Grafana dashboard, try our [Visualize Data with Grafana tutorial](/tutorials/services/visualize-data-grafana/).
+If you'd like to create your own customizable dashboard using the Viam TypeScript, continue with this tutorial.
+
+## Code your custom TypeScript dashboard
+
+The [Viam TypeScript SDK](https://ts.viam.dev/) allows you to build custom web interfaces to interact with your machines.
+For this project, you'll use it to build a page that displays air quality sensor data for a given location.
+You'll host the website locally on your personal computer, and view the interface in a web browser on that computer.
+
+As you'll find out in the [authentication step](#authenticate-your-code-to-your-viam-app-location), you can set each customer up with credentials to access the data from only their location, or you can create a dashboard showing data from all sensors in your entire organization.
+
+{{<imgproc class="aligncenter" src="/tutorials/air-quality-fleet/two-sensors.png" resize="x1000" declaredimensions=true alt="The air quality dashboard you'll build. This one has PM2.5 readings from two different sensor machines displayed, and a key with categories of air quality." style="max-width:500px" >}}
+
+### Set up your TypeScript project
+
+Complete the following steps on your laptop or desktop.
+You don't need to install or edit anything else on your machine's single-board computer (aside from `viam-server` which you already did); you'll be running the TypeScript code from your personal computer.
+
+1.  Make sure you have the latest version of [Node.JS](https://nodejs.org/en) installed on your computer.
+1.  Install the Viam TypeScript SDK by running the following command in your terminal:
+
+    ```sh {class="command-line" data-prompt="$"}
+    npm install --save @viamrobotics/sdk
+    ```
+
+1.  Create a directory on your laptop or desktop for your project.
+    Name it <file>aqi-dashboard</file>.
+
+1.  Create a file in your <file>aqi-dashboard</file> folder and name it <file>package.json</file>.
+    The <file>package.json</file> file holds necessary metadata about your project.
+    Paste the following contents into it:
+
+    ```json {class="line-numbers linkable-line-numbers"}
+    {
+      "name": "air-quality-dashboard",
+      "description": "A dashboard for visualizing data from air quality sensors.",
+      "scripts": {
+        "start": "esbuild ./main.ts --bundle --outfile=static/main.js --servedir=static --format=esm",
+        "test": "echo \"Error: no test specified\" && exit 1"
+      },
+      "author": "Viam Docs Team",
```

Comment:
Do we want to say anything about changing the author?

Reply:
Hmm I suppose it doesn't matter too much? I can't add a comment in the JSON of course but I can add a sentence just below saying change it to your name.